### PR TITLE
Add Pl Navitia provider (Municipal Transports of Poland)

### PR DESCRIPTION
**Pull request by mimi89999**

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -130,7 +130,7 @@ dependencies {
 	implementation "com.google.dagger:dagger:$daggerVersion"
 	kapt "com.google.dagger:dagger-compiler:$daggerVersion"
 
-	implementation('com.gitlab.opentransitmap:public-transport-enabler:9c909718') {
```
Comment (grote) [inline app/build.gradle]: Did you test this? Looks like you didn't update the dependency pinning. See https://github.com/grote/Transportr/pull/479#issuecomment-407740999 for more details!

Comment (mimi89999) [reply app/build.gradle]: @grote You removed verification in https://github.com/grote/Transportr/commit/4bd5924b65bafc4e18f582d6e6ae6fd81ee8d29d#diff-39e7d8c00954e920b98e7636f0ac30b2

Comment (mimi89999) [reply app/build.gradle]: Shall I add it back?

**Comment by mimi89999** [comment]
@grote I changed the commit, readded dependency pinning and signed CLA.

**Comment by grote** [comment]
Awesome! Thanks a lot! :smile: :bouquet:

**Comment by grote** [comment]
@cla-bot[bot] check

**Comment by grote** [comment]
I noticed the PNG logo too late. We are kind of not accepting those anymore. Would you mind replacing this with a VectorDrawable? See #238

**Comment by mimi89999** [comment]
The wiki said that PNG are OK. I will change that PNG for a drawable tomorrow.

**Pull request merged**